docs/src/content/docs/resources/community-content.mdx

```diff
@@ -119,6 +119,21 @@ Explore community-produced content maintained by Starlight users:
 		title="Add a reading progress indicator"
 		description="Find out how to add a horizontal reading progress bar to your Starlight site."
 	/>
+	<LinkCard
+		href="https://blog.trueberryless.org/blog/starlight-sidebar-whitespace/"
+		title="Make your Starlight sidebar items look better"
+		description="Using a little bit of CSS to adjust fonts, colours and more of the Starlight sidebar."
+	/>
+	<LinkCard
+		href="https://blog.trueberryless.org/blog/starlight-dropdown-and-list-together/"
+		title="Starlight Topics Dropdown on mobile, List on desktop"
+		description="Combine two plugins to have the best of both on desktop and mobile devices."
+	/>
+	<LinkCard
+		href="https://blog.trueberryless.org/blog/starlight-autogenerate-whole-sidebar/"
+		title="Convention for better Starlight sidebars with middleware"
+		description="Use middleware and your own convention to have better control of the order of sidebar items and group labels."
```

Comment:
I’m not very clear what “convention” means in this context, so maybe it’s not helping get the message across. Could just be edited down to something like this?
```suggestion
		description="Use middleware to take control of the order of sidebar items and group labels."
```

Reply:
Ahh, thanks for the explanation! I think maybe this would work well:

```suggestion
		description="Use route middleware to control sidebar ordering and group labels, inspired by Nuxt Content’s conventions."
```